plugin/harpoon.vim

```diff
@@ -34,7 +42,7 @@ fun! SetBuffer(ctrlId)
         return
     end
 
-    let g:win_ctrl_buf_list[a:ctrlId] = [b:terminal_job_id, nvim_win_get_buf(0)]
```

Comment:
So vim operates on simply the buffer handle and thats it?

Interesting.

Reply:
I didn't get the question.

Reply:
just looking at the code.  I notice that you use `bufnr()` to grab an id and to communicate with the terminal.  So vim terminal just uses buffer id to do communication?

Reply:
Yeah, it only need terminal buffer id to do communication
Vim doesn't use any kind of terminal id for communication